internal/proxy/proxy.go

```diff
@@ -747,6 +747,14 @@ type socketMount struct {
 	dialOpts []cloudsqlconn.DialOption
 }
 
+func getNetworkType(conf *Config, inst InstanceConnConfig) string {
```

Comment:
Let's make this `networkType`. Cf. https://go.dev/doc/effective_go#Getters

Reply:
Fixed. Thanks!